src/main/scala/com/typesafe/sbt/packager/docker/DockerSpotifyClientPlugin.scala

```diff
@@ -77,10 +77,11 @@ object DockerSpotifyClientPlugin extends AutoPlugin {
       def progress(message: ProgressMessage) = {
         Option(message.error()) match {
           case Some(error) if error.nonEmpty => log.error(message.error())
-          case _ => log.info(message.stream())
+          case _ if message.stream() != null => log.info(message.stream())
```

Comment:
I thought that it would be simpler than foreach, but i can change it to:
```scala
case _ => Option(message.stream()).foreach(log.info)
```

It that ok? :)